fakeredis/_server.py

```diff
@@ -878,7 +878,7 @@ def _scan(self, keys, cursor, *args):
         However, provided the database is not modified, every key will be
         returned exactly once.
         """
-        pattern = None
+        pattern = type = None
```

Comment:
I'm not a fan of squashing multiple assignments onto one line like this.
```suggestion
        pattern = None
        type = None
```

Reply:
Fixed.